packages/react-portal-compat/package.json

```diff
@@ -27,21 +27,16 @@
   },
```

Comment:
@Hotell Is there sense to bump this package directly to v9? Or use a different version (1.0.0)?

Reply:
good question 🤔

- from logical perspective I'd go with 1.x
- from "how we approach things related to v9" I'd go with 9 :D 

let's go with 9 ? 😅

Reply:
FYI: I also bumped it to be `-rc`.

Reply:
me gusta